rapids/rapids.sh

```diff
@@ -9,11 +9,11 @@ function get_metadata_attribute() {
 }
 
 readonly SPARK_VERSION_ENV=$(spark-submit --version 2>&1 | sed -n 's/.*version[[:blank:]]\+\([0-9]\+\.[0-9]\).*/\1/p' | head -n1)
-readonly DEFAULT_SPARK_RAPIDS_VERSION="0.3.0"
+readonly DEFAULT_SPARK_RAPIDS_VERSION="0.4.0"
 
 if [[ "${SPARK_VERSION_ENV}" == "3"* ]]; then
   readonly DEFAULT_CUDA_VERSION="10.2"
-  readonly DEFAULT_CUDF_VERSION="0.18"
+  readonly DEFAULT_CUDF_VERSION="0.18.1"
   readonly DEFAULT_XGBOOST_VERSION="1.3.0"
   readonly DEFAULT_XGBOOST_GPU_SUB_VERSION="0.1.0"
   # TODO: uncomment when Spark 3.1 jars will be released.
```

Comment:
Are we able to revert this now, as well as lines 40-48??

Reply:
Hi Brad, I don't think Spark use RAPIDS_VERSION at all, so 40-48 were never needed?

https://github.com/GoogleCloudDataproc/initialization-actions/blob/fbddbda8283869c4aad309c6956e41b0faa713ac/rapids/rapids.sh#L40-L48

Reply:
I removed the lines

Reply:
Ahh yes good catch, it relied on `DEFAULT_CUDF_VERSION`. Thanks for fixing!

Reply:
Hi Brad, now spark rapids work with spark 3.1, but XGBoost repo still has spark 3.0 in their maven URL, we cannot remove TODO comment just yet, but the cluster seems to be working. I Xgboost should work with spark 3.1, it is just for maven url

Reply:
Ok great, thank you for clarifying 😄